fvtest/compilertriltest/JitTest.hpp

```diff
@@ -40,7 +42,8 @@ class JitTest : public ::testing::Test
 
    static void SetUpTestCase()
       {
-      ASSERT_TRUE(initializeJit()) << "Failed to initialize the JIT.";
+      auto initSuccess = initializeJitWithOptions((char*)"-Xjit:acceptHugeMethods,enableBasicBlockHoisting,omitFramePointer,useIlValidator,paranoidoptcheck");
+      ASSERT_TRUE(initSuccess) << "Failed to initialize the JIT.";
```

Comment:
Do we _need_ `omitFramePointer`, or is this just to match the JitBuilder init code?